### ISSUE TITLE
from datetime import datetime is never given for datetime

### ISSUE THREAD

**Issue by paradoxxxzero** [alecthomas/importmagic#41]
I narrowed it down to the [fixup method] (https://github.com/alecthomas/importmagic/blob/master/importmagic/index.py#L245) but no real succes in fixing this aside for this really dirty hack: https://github.com/paradoxxxzero/importmagic/commit/0bfe7a9afaf8fc0d0b7bde5b9e88376b1fc03434

Any thoughts?

**Comment by alecthomas** [comment]
Can you check your index to see if that symbol is available?

**Comment by paradoxxxzero** [comment]
Yes it is, all works fine until the fixup method removes the variable
because it's the same as the module, which is what we want for
`datetime.datetime` but not for `datetime`.
In both of these cases, the fixup method is called twice, once with
`datetime`, None and once with `datetime`, `datetime`. However it always
returns `datetime`, None which is wrong for the latter case.

I hope I'm being clear enough :)

Btw Thanks for this awesome project, that I now use daily in
http://github.com/Kozea/wdb and Atom
https://github.com/paradoxxxzero/atom-python-import-magic

Le sam. 17 sept. 2016 à 09:24, Alec Thomas <notifications@github.com> a
écrit :

**Comment by kbauer** [comment]
Ran into the same issue today with ``pprint.pprint``, which I normally import as

    from pprint import pprint

In a similar manner, completing for "sin" offers "from numpy.ma.core import sin" but not the more sensible "from numpy import sin".
